Readability.js

```diff
@@ -1737,7 +1763,7 @@ Readability.prototype = {
         var contentLength = this._getInnerText(node).length;
 
         var haveToRemove =
-          (img > 1 && img > p && !this._hasAncestorTag(node, "figure")) ||
+          (img > 1 && p / img < 0.5 && !this._hasAncestorTag(node, "figure")) ||
```

Comment:
```
Oh, maybe I do know.... before this change, was all the whitespace between nodes being converted to <p> tags which then added weighting to the container which led to it being kept, given that it also has images and we have some logic about removing image containers without too much text and so on?
```
Thanks a lot. You're right. I've updated patch to fix it. How about this, which means number of `<img>` node need to be bigger enough than `p` node?